src/dune_config/config.ml

```diff
@@ -116,7 +116,10 @@ let background_digests =
   let t =
     { name = "background_digests"
     ; of_string = Toggle.of_string
-    ; value = background_default
+    ; value =
+        (match Platform.OS.value with
+        | Linux -> `Enabled
```

Comment:
Why did you disable it on Mac and the rest of the unixes?

Reply:
Oh, I remember that we have that fork bug as well.

Reply:
I can introduce a `Toggle.and_`  to express that as `background_default and not windows` if you think it's more explicit.